fuzzers/fuzz.cpp

```diff
@@ -0,0 +1,241 @@
+
```

Comment:
Did you copy these functions from the source code? If so, they are static copies and will not track changes if the upstream functionality is changes. Please link in or include the functions rather than copying them. Also, please rename this from `fuzz.cpp` to something that describes what's being fuzzed? Also, I don't think using std::cout is the best way to read a file in. Maybe you could use libFuzzer here? I also recommend not having any std:cout for errors to improve fuzzing speed.

Reply:
Yes it is an example code reading from stdin,  most of the functions are printing stuff so I just refactored the harness to be fuzz_3mfReader.cpp.
std::cout calls deleted as well.